libs/packages/sam-formly/src/lib/formly/wrappers/form-field.wrapper.ts

```diff
@@ -12,6 +12,9 @@ import { FieldWrapper } from '@ngx-formly/core';
   <span>{{ to.label }}</span>
     <span *ngIf="to.required && to.hideRequiredMarker !== true">*</span>
   </label>  
+
+  <label *ngIf="to.hideLabel" [style.display]="to.hideLabel ? 'none' : ''" [attr.for]="id">{{ to.label }}</label>
```

Comment:
What is the use case for removing the label? How can it be 508 compliant if we provide this option?